build.gradle

```diff
@@ -406,7 +406,8 @@ check.dependsOn installDist
 
 model {
     components {
-        VectorLoglessPairHMM(NativeLibrarySpec) {
+        if (System.properties["os.arch"] != "ppc64le") {
+	VectorLoglessPairHMM(NativeLibrarySpec) {
```

Comment:
Could you replace this tab with spaces.  Probably best to set you're IDE to replace tabs with spaces  when working on gatk.